tests/features/fixtures/mazerunner/src/main/java/com/bugsnag/android/mazerunner/MainActivity.kt

```diff
@@ -14,6 +15,8 @@ import com.bugsnag.android.mazerunner.scenarios.Scenario
 
 class MainActivity : Activity() {
 
+    private val apiKeyKey = "BUGSNAG_API_KEY"
```

Comment:
```suggestion
    private val apiKey = "BUGSNAG_API_KEY"
```

Reply:
Although it looks like a typo I think this is actually correct - "BUGSNAG_API_KEY" is the key for getting the apiKey.

Reply:
Ah - I'd assumed that this was a variable that holds the API key. Perhaps something like the following would make it more clear that it's a constant:

`private const val EXTRA_API_KEY = "BUGSNAG_API_KEY"`